src/pathfinding/PathfinderResultPrinter.cpp

```diff
@@ -57,6 +57,47 @@ std::vector<std::pair<double, double>> PathfinderResultPrinter::GetVector(HexPla
   return pathResult;
 }
 
+std::vector<std::pair<double, double>> PathfinderResultPrinter::GetHardcoded(std::string test_name) {
+
+  std::vector<std::pair<double, double>> pathResult;
+
+  if (!test_name.compare("Oct")) {
+      pathResult.push_back(std::make_pair(-123.160124, 49.275202));
+      pathResult.push_back(std::make_pair(-123.307560, 49.212129));
+      pathResult.push_back(std::make_pair(-123.494778, 49.059004));
+  }
+
```

Comment:
Could you add a print saying "Warning: hardcoded test_name " << test_name << " could not be found." if it's not found? Nice safety check :)